HOWTO.md

```diff
@@ -80,7 +95,16 @@ $ aws cloudformation deploy \
     --capabilities CAPABILITY_IAM
 ```
 
-Refer to the [documentation](http://docs.aws.amazon.com/cli/latest/reference/cloudformation/deploy/index.html) for more details.
+Or using `aws-sam-cli` 
+
+```bash
+$ sam publish \
+    --template-file /path_to_template/packaged-template.yaml \
+    --region AWS_REGION
+    --capabilities CAPABILITY_IAM
```

Comment:
The `sam publish` command is VERY different than the `sam deploy` command. Best not to get those mixed up in the basic how-to documentation.
`sam deploy` is an alias for `aws cloudformation deploy`.
`sam publish` publishes the app to the Serverless Application Repository.